python/rmm/_lib/lib.pyx

```diff
@@ -197,25 +197,24 @@ def rmm_alloc(size, stream):
     return int(c_addr)
 
 
-cdef void c_free(void *ptr, cudaStream_t stream) except *:
+cdef rmmError_t c_free(void *ptr, cudaStream_t stream,
+                       const char* file=NULL, unsigned int line=0) except *:
     """
     Deallocates ptr, which was allocated using rmmAlloc by calling the librmm
     functions via Cython
     """
-    cdef caller_pair tmp_caller_pair = _get_caller()
-    cdef const char* file = tmp_caller_pair.first
-    cdef unsigned int line = tmp_caller_pair.second
+    cdef rmmError_t rmm_error
 
     # Call RMM to free
     with nogil:
         rmm_error = rmmFree(
             <void *>ptr,
             <cudaStream_t>stream,
-            <const char*>file,
-            <unsigned int>line
+            file,
+            line
```

Comment:
Either add back in the casts to be consistent or remove them for `ptr` and `stream` as well.